modules/account-lib/src/coin/hbar/utils.ts

```diff
@@ -182,3 +183,35 @@ export function removePrefix(prefix: string, key: string): string {
   }
   return key;
 }
+
+/**
+ * Validate if a hedera encoded transaction was signed by a certain
+ * signer and if the signature is the same
+ *
+ * @param {string} encodedTx a hedera encoded transaction
+ * @param {KeyPair} key the signer's keypair
+ * @param {string} signature the signature to be compared
+ * @returns {boolean} the result of the validation
+ */
+export function validate(encodedTx: string, key: KeyPair, signature: string): boolean {
```

Comment:
We should probably put this in the `Transaction` object since we already have that info there

Reply:
That sounds good, w'll go for this approach